chimney/src/main/scala/io/scalaland/chimney/internal/utils/MacroUtils.scala

```diff
@@ -12,6 +12,7 @@ trait MacroUtils extends CompanionUtils {
     def toNameConstant: Constant = Constant(n.decodedName.toString)
     def toNameLiteral: Literal = Literal(toNameConstant)
     def toSingletonTpe: ConstantType = c.internal.constantType(toNameConstant)
+    def toCanonicalName: String = n.toString
```

Comment:
Just wanted to explain this bit. I noticed, there's a demand for case-insensitive match (#193) . The idea behind this method is to simplify that feature development in the future